### PR TITLE
Update markup extension

### PR DESCRIPTION
**Pull request by krOoze**
- resolves new asciidoctor warnings
- obviates the need to escape ->

### CODE REVIEW

**Comment by oddhack** [comment]
Can you tweak the example in the markup section of the style guide to match?

**Comment by krOoze** [comment]
@oddhack Right, done.

**Comment by oddhack** [comment]
Is this dependent on asciidoctor 2.0 internals or something like that? When I try building I get

> asciidoctor: FAILED: /home/tree/git/Vulkan-Docs/vkspec.txt: Failed to load AsciiDoc document - undefined method `create_inline_pass' for #<PnameInlineMacro:0x000055f390278ce0>
> Did you mean?  create_inline

**Comment by krOoze** [comment]
@oddhack It's possible. I am not in the business of testing stuff for the whole history of versions. I just use the most recent one on `ruby-gems`.

`create_inline_pass` is I think relatively simple wrapper for `create_inline`. Assuming your favorite version has `create_inline` (that works right) it could be made to work too...

**Comment by krOoze** [comment]
@oddhack extension re-processing of substitutions require 2.0. Which is much cleaner than to use the tags.

Meanwhile I pushed a changes that should work with the old version.

File: style/extensions.txt [hunk 1]
```diff
@@ -903,12 +903,12 @@ In this case, the type's etext:Vk prefix is replaced with the enum prefix
 etext:VK_OBJECT_TYPE_, and the rest of the handle name is converted as
 described in that section.
 
-[source,asciidoc]
-.Conversion of Handle to VkObjectType Examples:
-----
- VkSurfaceKHR                  -> VK_OBJECT_TYPE_SURFACE_KHR
- VkDescriptorUpdateTemplateKHR -> VK_OBJECT_TYPE_DESCRIPTOR_UPDATE_TEMPLATE_KHR
-----
+[literal, subs="+replacements"]
```
[the remark this answers is not in the document]
Comment (krOoze) [reply style/extensions.txt]: Just collateral damage when searching for the `->`.
This markup shown C arrows when it meant to use unicode arrows,

**Pull request merged**